### ISSUE TITLE
Spotlight effect for mouseover

### ISSUE THREAD

**Issue by Backupiseasy** [Backupiseasy/ThreatPlates#31]
I appreciate you coming in and getting threatplates back into action!

During the period of time I was forced to use the base tidyplates addon, the one feature that I really have grown somewhat attached to is the "Spotlight" feature. Mostly for Mouseover and Casting units.

Would it be possible to add this feature to ThreatPlates? Perhaps adding a checkbox and slider for the following options:

Custom Mouseover Alpha/ Scale

Custom Casting Alpha / Scale

Basically like the current Target/No-Target custom Alpha/Scale exist, but for mouseover and/or casting units.
[Source: [https://wow.curseforge.com/addons/tidy-plates-threat-plates/?comment=258](https://wow.curseforge.com/addons/tidy-plates-threat-plates/?comment=258)]
